components/content/content-editor/plugins/update-incrementer-plugin.ts

```diff
@@ -22,6 +22,10 @@ export class UpdateIncrementerPlugin {
 		}
 		// If anything in the editor changes (content or selection), make sure we increment, so we can for example reposition controls.
 		this.appEditor.stateCounter++;
+		if (this.appEditor.canShowMentionSuggestions > 0) {
+			this.appEditor.canShowMentionSuggestions--;
```

Comment:
please comment on this, i dont understand what this variable is even supposed to be doing.